### ISSUE TITLE
My thing is blocked, and keys won't show up when it was unblocked.

### ISSUE THREAD

**Issue by Sillygooberton** [BOLL7708/OpenVR2Key#95]
### Current behavior

It is currently unblocked and I don't understand any of this openvr2key stuff.

### Steps to reproduce the behavior

I don't know

### Expected behavior

I expect that when I first opened up openvr2key that if I just pressed a button and boom it works but no it dosen't.

### Operating system version

Windows 10

### SteamVR version

Idk the latest?

### Application version

Latest

### VR HMD

Oculus quest 2

### VR hardware

Ocukus quest 2 plain no add-ons

### Software to control

Gorrila tag (I haven't gotten openvr2key to work outside of it

### Game

Gorrila tag (I haven't gotten openvr2key to work outside of it

### Additional information

None

**Comment by BOLL7708** [comment]
Unblocked just means there has not been any detected input yet. This works with SteamVR, it won't do anything if you don't have SteamVR running at the same time.

**Comment by Sillygooberton** [comment]
It won't say there are any keys being pressed when I do press the buttons and its blocked now.

**Comment by BOLL7708** [comment]
1. Are you running SteamVR
2. Does the application say it is connected to SteamVR?
3. What bindings do you have set for the application in the SteamVR controller bindings?

If nothing else, provide a screenshot of the application in this thread.
